tgui/packages/tgui/interfaces/Aquarium.tsx

```diff
@@ -13,98 +18,338 @@ import { Window } from '../layouts';
 
 type Data = {
   temperature: number;
-  fluid_type: string;
+  fluidType: string;
   minTemperature: number;
   maxTemperature: number;
   fluidTypes: string[];
-  contents: { ref: string; name: string }[];
-  allow_breeding: BooleanLike;
-  feeding_interval: number;
+  fishData: fishData[];
+  propData: propData[];
+  allowBreeding: BooleanLike;
+  feedingInterval: number;
+  heartIcon: string;
+  heartIconState: string;
+  heartEmptyIconState: string;
+};
+
+type fishData = {
```

Comment:
```suggestion
type FishData = {
```
pascal case for types